libs/consts/src/artifact.ts

```diff
@@ -57,3 +57,5 @@ export const artMaxLevel: Record<RarityKey, number> = {
   4: 16,
   5: 20,
 } as const
+
+export const allArtifactRarityKeys = [5, 4, 3] as const

```

Comment:
This is a GO-only field because only GO limits artifact rarity. I would prefer if this stays in the front-end only side of the code